src/hosting.js

```diff
@@ -7,8 +7,8 @@ import hostingAPI from "./hosting-api.js";
  * @param {string|array} domain - The domain to check, or an array of domains to be checked.
  * @returns {boolean|array} - A boolean if a string was provided, or an array of booleans if an array of domains was provided.
  */
-function check(domain) {
-  return hostingAPI.check(domain);
+function check(domain, comment) {
```

Comment:
I think the function signature would need the updated comment here as well. The same comment we use elsewhere ought to work.

```
* @param {string} comment - Optional. The app, site, or organisation that is making the request.
```